gooey/gui/processor.py

```diff
@@ -64,11 +66,21 @@ def _forward_stdout(self, process):
         Reads the stdout of `process` and forwards lines and progress
         to any interested subscribers
         '''
+        _start_time = get_current_time()
         while True:
             line = process.stdout.readline()
             if not line:
                 break
             _progress = self._extract_progress(line)
+
+            # For displaying elapsed & estimated remaining time
+            if _progress is not None and _progress > 0 and self.show_time_remaining:

```

Comment:
See other comments. We should rip this stuff out.

Reply:
Resolved 😄